src/server/client.rs

```diff
@@ -102,6 +102,14 @@ pub async fn client_rpc_loop<
                         compute_job_detail(&state_ref, msg.job_id, msg.include_tasks)
                     }
                     FromClientMessage::Stats => compose_server_stats(&state_ref, &tako_ref).await,
+                    FromClientMessage::StartCliDashboard => {
```

Comment:
I don't think that we want to run a dashboard loop on the server. It should run on the client instead. @spirali?

Reply:
That was a mistake :) 
I've pushed my latest experiment where I was trying to get the data from the workers and bring it to the UI. It will be cleaned and done properly but I wanted to make something to start with.